terraform/modules/marklogic/README.md

```diff
@@ -1,4 +1,12 @@
 To use this module you need to manually create two AWS secrets:
 
 * `ml-license-${var.environment}` with a value of the form `{"licensee":"foo","license_key":"bar"}`
-* `ml-admin-user-${var.environment}` with a value of the form `{"username":"foo","password":"bar"}`
\ No newline at end of file
+* `ml-admin-user-${var.environment}` with a value of the form `{"username":"foo","password":"bar"}`
+
+You can manually ssh onto a MarkLogic instance and check the service logs with:
+* Port forwarding: `ssh hugeme@$(terraform output -raw bastion_dns_name) -L localhost:9001:instance.ip.here:22`
+* SSH (with private key taken from module.marklogic.ml_ssh_private_key): `ssh -i ~/.ssh/privatekey ec2-user@localhost -p 9001`
+* On the instance: `journalctl -u MarkLogic`
```

Comment:
If these are sequential steps, i.e. you have to do all three in that order, I think they should be a numbered list, not a bulleted one. Presumably also the username should be more generic, and not Hugh's domain username?